### DISCUSSION TITLE
Cant access the vpnHood in our country

### DISCUSSION THREAD

**Discussion by Majid-Sadeghian** [vpnhood/VpnHood#142] [Q&A] [answered]
Dear VpnHood team
Thanks for your perfect app I've been using it for a while and it's excellent.
But these days, our government has started to heavily censor and restrict social media. 
We can't connect to most VPNs and that includes VpnHood. The Proton VPN guys run a protocol on Android and call it Stealth. 
This is the only way we can access the internet in our country today. I want to ask if you can run something like Stealth protocol on VpnHood.
This will help my people a lot.
Thanks for your excellent services.

**Comment by trudyhood** [comment, marked as answer]
Hi,
Thank you for trying VpnHood. It is good that companies are developing undetectable protocols like Stealth. The VpnHood protocol is undetectable with zero fingerprints.
You said that you couldn't connect to VpnHood. It is normal if you try our **public servers**. At the moment, many users have connected to VpnHood's private servers. I recommend you set up a **private server** and enjoy it. If someone could detect VpnHood on your private server, share it with us so we can work on it.

**Reply by Majid-Sadeghian** [reply]
Thank you for your response
I read this wiki page for private servers:
https://github.com/vpnhood/VpnHood/wiki/VpnHood-Access-Server
But currently, I don't have VPS or any other cloud service.
I wish VpnHood could offer undetectable protocols soon.
But thank you for making and developing such a great app like VpnHood. I like that.

**Comment by trudyhood** [comment]
> I wish VpnHood could offer undetectable protocols soon.

VpnHood protocol is already undetectable.

**Reply by zoghal** [reply]
I confirm this claim from inside Iran✌️🥰

**Comment by mahdisarlak** [comment]
Hello, I also had the same problem, but the reason for this is the UDP service filter and limitation in our country, you must use VPNHOOD without UDP option.

If you need a more detailed review, you can count on me

**Reply by bobvhood** [reply]
> Hello, I also had the same problem, but the reason for this is the UDP service filter and limitation in our country, you must use VPNHOOD without UDP option.
> 
> If you need a more detailed review, you can count on me

Hi @mahdisarlak 
The UDP protocol is disabled by default.

**Comment by rezarms** [comment]
@mahdisarlak  Were you able to make it work from Iran?

**Reply by zoghal** [reply]
اقا کار می‌کند

**Reply by rezarms** [reply]
It doesn't work for Irancel provider.

**Reply by zoghal** [reply]
Unfortunately yes! 
Irancell operator has been able to identify and block anonymous traffic for some time.
I wish it was possible to hide the server IP behind CDN?!

@trudyhood Is it possible in vpnhood?👆

**Comment by trudyhood** [comment]
We are going to release a version that users or server owners can disable UDP at all. It is now on one of our top priority list.

**Reply by Majid-Sadeghian** [reply]
> We are going to release a version that users or server owners can disable UDP at all. It is now on one of our top priority list.

Thank you
I hope we can use VPNHood again as soon as possible.

**Comment by ahadvirus** [comment]
Thanks for your support.
As mentioned by @rezarms in the discussion one of the IPS detected your project and disturbed it for its clients.
If I want to be specific about the situation of your project at the IPS, I have to tell you, we can be connected to the server but when trying to transfer the data causes it to disconnect.
In end, I have to say we can only use the TCP protocol.

**Reply by MohMehrnia** [reply]
I run private server but is not work, (irancell isp)

**Reply by trudyhood** [reply]
Are you sure that your new server IP is not already blocked?

> I run private server but is not work, (irancell isp)

**Reply by zoghal** [reply]
I also confirm this.
  Of course, I think the main reason is that Irancell currently allows the transfer of packages based on the white list. Because this operator has recently applied strange and unprecedented restrictions.

**Reply by trudyhood** [reply]
> I also confirm this. Of course, I think the main reason is that Irancell currently allows the transfer of packages based on the white list. Because this operator has recently applied strange and unprecedented restrictions.

If you are on the whitelist, the only solution is to use a personal local server forwarder. This involves buying a server in your own country and running a port forwarder to forward the port to the VpnHood server. However, it's important to ensure that you don't share this service publicly because the responsibility for the server lies with the owner.
